### ISSUE TITLE
Improve the gradle plugin, should support arbitrary vmf source folders

### ISSUE THREAD

**Issue by miho** [miho/VMF#11]

**Comment by miho** [comment]
The property

    ext.vmfPluginIntelliJIntegration = true

enables additional intellij support (vmf folders are added ass source root and generated src dirs are marked as being generated, i.e., intellij will warn whenever a change is made to the generated file)

<img width="366" alt="screenshot 2019-01-23 23 40 14" src="https://user-images.githubusercontent.com/225266/51642093-6ac70b00-1f68-11e9-9642-b91bd2d2ac6b.png">

<img width="637" alt="screenshot 2019-01-23 23 33 43" src="https://user-images.githubusercontent.com/225266/51642083-5f73df80-1f68-11e9-9d38-f93ee6ddfc3c.png">

**Comment by miho** [comment]
If idea has problems resolving classes in the vmf folder, one can help it by declaring the dependency manually, which will also enable full intellisense:

```gradle    
idea {
    module {
        dependencies {
            compile group: 'eu.mihosoft.vmf', name: 'vmf', version: commonProps.get("publication.version")
        }
    }
}
```

**Comment by miho** [comment]
This feature has been merged mack to master and is ready to ship in 0.2.
